components/Navbar.tsx

```diff
@@ -23,7 +23,10 @@ const Navbar = () => {
   const [searchOpen, setSearchOpen] = useState(false)
   const openSearchBox = () => setSearchOpen(true)
 
-  useHotkeys(`${os === 'mac' ? 'cmd' : 'ctrl'}+k`, openSearchBox)
+  useHotkeys(`${os === 'mac' ? 'cmd' : 'ctrl'}+k`, e => {
+    openSearchBox()
+    e.preventDefault()
```

Comment:
```suggestion
    e.preventDefault()
    openSearchBox()
```

should be on top, right? or does it not matter that much?

Reply:
`openSearchBox()` does not take the event as args plus JS is basically single-threaded, so they should have the same effect, though the suggested one may be a little more gentle.

Reply:
Oh never mind, I believe I've seen both ways of writing this. Merging as is.